content/docs/concepts/delivery/index.md

```diff
@@ -11,7 +11,7 @@ Keptn allows you to declaratively define *multi-stage delivery* workflows by def
 
 ## Shipyard for Delivery Declaration
 
-The definition is manifested in the [shipyard](../1.0/reference/files/shipyard] file that defines a task sequence for delivery. It can hold multiple stages, each with a dedicated *deployment strategy*, *test strategy*, as well as a *remediation strategy*. Following this declarative approach, there is no need to write imperative pipeline code. Keptn takes the shipyard file and creates a multi-stage workflow each stage having a deployment strategy (e.g., blue/green), testing strategy (e.g., functional tests or performance tests), and an optional automated remediation strategy for triggering self-healing actions. To learn more about a shipyard configuration, please continue [here](../../0.15.x/continuous_delivery/multi_stage/).
+The definition is manifested in the [shipyard](../1.0/reference/files/shipyard] file that defines a task sequence for delivery. It can hold multiple stages, each with a dedicated *deployment strategy*, *test strategy*, as well as a *remediation strategy*. Following this declarative approach, there is no need to write imperative pipeline code. Keptn takes the shipyard file and creates a multi-stage workflow each stage having a deployment strategy (e.g., blue/green), testing strategy (e.g., functional tests or performance tests), and an optional automated remediation strategy for triggering self-healing actions. To learn more about a shipyard configuration, please continue [here](../../{{< param "version" >}}/continuous_delivery/multi_stage/).
```

Comment:
```suggestion
The definition is manifested in the [shipyard](../1.0/reference/files/shipyard] file that defines a task sequence for delivery. It can hold multiple stages, each with a dedicated *deployment strategy*, *test strategy*, as well as a *remediation strategy*. Following this declarative approach, there is no need to write imperative pipeline code. Keptn takes the shipyard file and creates a multi-stage workflow each stage having a deployment strategy (e.g., blue/green), testing strategy (e.g., functional tests or performance tests), and an optional automated remediation strategy for triggering self-healing actions. 
```
All this info about continuous delivery needs to be purged and I haven't gotten to it.  But we can at least remove this statement -- the shipyard reference page (referenced in the first sentence of this section) is the most comprehensive summary of shipyard.